docker-compose.yaml

```diff
@@ -7,6 +7,8 @@ services:
   yarn-resourcemanager:
     image: ghcr.io/kbase/cdm-prototype-yarn:pr-8
     container_name: yarn-resourcemanager
+    # Images from the ghcr.io/kbase registry are exclusively available for Linux/AMD64 platforms.
+    platform: linux/amd64
```

Comment:
Does this fix the mac issues?

Reply:
No. Without this, docker compose won't run on Mac machines since it cannot find the arm64 version. I used to manually modify docker-compose file before running it.